src/components/[guild]/RolePlatforms/components/AddRoleRewardModal/components/AddContractCallPanel/components/CreateNftForm/CreateNftForm.tsx

```diff
@@ -145,273 +149,282 @@ const CreateNftForm = ({ onSuccess }: Props) => {
 
   return (
     <FormProvider {...methods}>
-      <Stack spacing={8}>
-        <Text colorScheme="gray" fontWeight="semibold">
-          Create an NFT that members will be able to mint if they satisfy the
-          requirements you'll set. Claiming can take place through your Guild page or
-          a fancy auto-generated minting page!
-        </Text>
-
-        <Grid w="full" templateColumns="repeat(3, 1fr)" gap={8}>
-          <GridItem colSpan={{ base: 3, sm: 1 }}>
-            <ImagePicker />
-          </GridItem>
-
-          <GridItem colSpan={{ base: 3, sm: 2 }}>
-            <Stack spacing={6}>
-              <HStack alignItems="start">
-                <FormControl isInvalid={!!errors?.name}>
-                  <FormLabel>Name</FormLabel>
+      <ModalHeader>Create NFT</ModalHeader>
```

Comment:
This breaks the styling in the "+ Add reward" modal (guild page), so let's keep the old CreateNftForm component and just wrap it in a modal in the create guild flow.